### PR TITLE
1) Update to ex-db-common 9

### LINKED ISSUE #16: Update to db-ex-common-9

### PR DESCRIPTION
**Pull request by pivnicek**
fixes #16

### CODE REVIEW

File: tests/Keboola/DbExtractor/AbstractRedshiftTest.php [hunk 1]
```diff
@@ -24,7 +25,7 @@ public function setUp()
         if (!defined('APP_NAME')) {
             define('APP_NAME', 'ex-db-redshift');
```
Comment (MiroCillik) [inline tests/Keboola/DbExtractor/AbstractRedshiftTest.php]: Would also remove this `APP_NAME` constant. Perhaps define it in some class as public static const?

Comment (MiroCillik) [reply tests/Keboola/DbExtractor/AbstractRedshiftTest.php]: How are we dealing with it in other db extractors? :)

**Review by MiroCillik — CHANGES_REQUESTED** [review]
Remove the APP_NAME constant, other than that I think it's fine.

File: tests/Keboola/DbExtractor/AbstractRedshiftTest.php (hunk 1 above)
Comment (pivnicek) [reply tests/Keboola/DbExtractor/AbstractRedshiftTest.php]: ah yes!  I missed that one, yes, they should all be removed.

**Comment by pivnicek** [comment]
Thanks @MiroCillik ! updated

**Review by MiroCillik — APPROVED** [review]

**Pull request merged**